pkg/model/sdk_api.go

```diff
@@ -327,9 +327,18 @@ func (a *SDKAPI) GetServiceFullName() string {
 	return a.API.Metadata.ServiceFullName
 }
 
-// APIInterfaceTypeName returns the name of the aws-sdk-go primary API
+// ClientInterfaceTypeName returns the name of the aws-sdk-go primary API
 // interface type name.
-func (a *SDKAPI) APIInterfaceTypeName() string {
+func (a *SDKAPI) ClientInterfaceTypeName() string {
+	if a == nil || a.API == nil {
+		return ""
+	}
+	return fmt.Sprintf("%s%s", a.API.StructName(), "API")
```

Comment:
Should this be `a.API.InterfaceName()` by default?

Reply:
There is no `a.API.InterfaceName` :/ . However there a `a.API.InterfacePackageName` but it return the package name that defines the interface.. which is not what we need here.

Reply:
Yeah, looking in the upstream code they indeed use `StructName` ... :/